lib/fortnox/api/repositories/base.rb

```diff
@@ -40,10 +40,7 @@ def self.set_headers(headers = {})
             options[:headers] = provided_headers
             options[:base_uri] ||= base_url
 
-            if Fortnox::API.debugging
-              Fortnox::API.logger.debug("path: #{path.inspect}")
-              Fortnox::API.logger.debug("options: #{options.inspect}")
-            end
+            debug_log_request(path, options) if Fortnox::API.debugging
```

Comment:
Added some debug logging when debugging this error. Moved it outside this method to fix Rubocop violation (too many lines in the method).

Reply:
I'm sorry ... but why on earth do you do: `debug_log_request(path, options) if Fortnox::API.debugging` instead of just implementing `debug_log_request` as:
```
def debug_log_request(path, options)
  return unless Fortnox::API.debugging
  Fortnox::API.logger.debug("path: #{path.inspect}")
  Fortnox::API.logger.debug("options: #{options.inspect}")
end

Reply:
Sure, I see now why your proposal is better than the current solution :D I'll change it 😄